### PR TITLE
Format errors on the web.

### PR DESCRIPTION
**Pull request by skzhang1**
## Why are these changes needed?
Format errors on the web. 

## Related issue number
https://github.com/microsoft/FLAML/issues/854

## Checks

- [x] I've used [pre-commit](https://microsoft.github.io/FLAML/docs/Contribute#pre-commit) to lint the changes in this PR, or I've made sure [lint with flake8](https://github.com/microsoft/FLAML/blob/816a82a1155b4de4705b21a615ccdff67c6da379/.github/workflows/python-package.yml#L54-L59) output is two 0s.
- [x] I've included any doc changes needed for https://microsoft.github.io/FLAML/. See https://microsoft.github.io/FLAML/docs/Contribute#documentation to build and test documentation locally.
- [] I've added tests (if relevant) corresponding to the changes introduced in this PR.
- [x] I've made sure all auto checks have passed.

### CODE REVIEW

**Comment by qingyun-wu** [comment]
Thanks Shaokun. Can you, by the way, also fix the problem mention in this issue: https://github.com/microsoft/FLAML/issues/854

File: flaml/tune/tune.py [hunk 1]
```diff
@@ -399,14 +399,14 @@ def easy_objective(config):
             - "tolerances"(optional): a dictionary to specify the optimality tolerances on objectives. The keys are the
```
Comment (sonichi) [inline flaml/tune/tune.py:399]:
```suggestion
            - "tolerances" (optional): a dictionary to specify the optimality tolerances on objectives. The keys are the
```

File: flaml/tune/tune.py [hunk 2]
```diff
@@ -399,14 +399,14 @@ def easy_objective(config):
...
-            lexico_objectives = {
-                "metrics": ["error_rate", "pred_time"],
-                "modes": ["min", "min"],
-                "tolerances": {"error_rate": 0.01, "pred_time": 0.0},
-                "targets": {"error_rate": 0.0},
-            }
-            ```
+    ```python
+    lexico_objectives = {
+        'metrics': ["error_rate", "pred_time"],
```
Comment (sonichi) [inline flaml/tune/tune.py:404]: Why changing " into '?

**Review by jtongxin — APPROVED** [review]

**Comment by skzhang1** [comment]
> Thanks Shaokun. Can you, by the way, also fix the problem mentioned in this issue: #854

Sure. fixed

**Review by sonichi — COMMENTED** [review]
fix lint issue: https://github.com/microsoft/FLAML/actions/runs/3735269251/jobs/6338310362#step:10:15

**Comment by sonichi** [comment]
> fix lint issue: https://github.com/microsoft/FLAML/actions/runs/3735269251/jobs/6338310362#step:10:15

Did you see this comment?

**Comment by skzhang1** [comment]
> > fix lint issue: https://github.com/microsoft/FLAML/actions/runs/3735269251/jobs/6338310362#step:10:15
> 
> Did you see this comment?

Sorry for the late response. I can not reproduce this error with flake8 locally and I do not find the reason now. I will try to fix it tomorrow.

File: flaml/automl/automl.py [hunk 3]
```diff
@@ -2816,10 +2816,10 @@ def is_to_reverse_metric(metric, task):
...
             logger.warning(
                 "No search budget is provided via time_budget or max_iter."
-                " Training only one model per estimator."
-                " Zero-shot AutoML is used for certain tasks and estimators."                
-                " To tune hyperparameters for each estimator,"
-                " please provide budget either via time_budget or max_iter."
+                "Training only one model per estimator."
+                "Zero-shot AutoML is used for certain tasks and estimators."
+                "To tune hyperparameters for each estimator,"
+                "please provide budget either via time_budget or max_iter."
```
Comment (sonichi) [inline flaml/automl/automl.py:2822]: The space in the beginning is needed. Otherwise there won't be spaces between the sentences.

Comment (skzhang1) [reply flaml/automl/automl.py]: I understand. Fixed. Since I can not automatically lint it using pre-commit, I suspect need to remove it.

**Review by qingyun-wu — APPROVED** [review]

**Pull request merged**